### PR TITLE
KT-20591 Show annotations in parameter info

### PR DESCRIPTION
**Pull request by shiraji**
#KT-20591 Fixed

https://youtrack.jetbrains.com/issue/KT-20591

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinFunctionParameterInfoHandler.kt [hunk 1]
```diff
@@ -254,6 +254,10 @@ abstract class KotlinParameterInfoWithCallHandlerBase<TArgumentList : KtElement,
         return buildString {
             if (named) append("[")
 
+            parameter.annotations.getAllAnnotations().forEach {
+                it.annotation.fqName?.shortName()?.asString()?.let { append("@$it ") }
```
Comment (semoro) [inline idea/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinFunctionParameterInfoHandler.kt]: Please, extract fqName to variable to reduce safe-call count

Comment (shiraji) [reply idea/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinFunctionParameterInfoHandler.kt]: How about this?

```kotlin
it.annotation.fqName?.let { append("@${it.shortName().asString()} ") }
```

Comment (semoro) [reply idea/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinFunctionParameterInfoHandler.kt]: Cool!

**Comment by shiraji** [comment]
There you go. Thanks!

**Comment by semoro** [comment]
Thanks! 🙂

**Pull request merged**